ru/lessons/basics/control-structures.md

```diff
@@ -127,3 +128,53 @@ iex> cond do
 ...> end
 "Catch all"
 ```
+
+## `with`
+
+Специальная форма `with` может пригодиться при использовании вложенных операторов `case` или в ситуациях, которые не могут быть просто слиты воедино. `with` состоит из ключевого слова, генераторов и, в конце, выражения.
```

Comment:
Your suggestion is really good in my opinion, @brain-geek . Thanks a lot!
I've just updated it and rebased the branch a moment ago.